libmachine/mcnutils/crc.go

```diff
@@ -229,8 +231,21 @@ func (b *B2dUtils) CopyDiskToMachineDir(diskURL, machineName string) error {
 
 	// By default just copy the existing "cached" iso to the machine's directory...
 	if diskURL == "" {
-		log.Infof("Copying %s to %s...", b.path(), machineIsoPath)
+		log.Infof("\n\nCopying %s to %s...\n\n\n", b.path(), machineIsoPath)
 		return CopyFile(b.path(), machineIsoPath)
+	} else if strings.HasPrefix(diskURL, "file:") {
+		// Local disk image so we can avoid copying by making use of backing image feature of qcow2
+		components := strings.Split(diskURL, "://")
+		if len(components) < 2 {
+			return fmt.Errorf("Invalid URL: %s", diskURL)
+		}
+		diskPath := components[1]
+		log.Infof("Create disk image at %s (backing store: %s)", machineIsoPath, diskPath)
+		command := exec.Command("qemu-img", "create", "-b", diskPath, "-f", "qcow2", machineIsoPath)
+		output, err := command.CombinedOutput()
+		log.Debugf("Output from `%s` command:\n%s", command.String(), output)
```

Comment:
```
if err != nil {
    log.Warnf("Error occurred running command: %s", err)
}
```

Reply:
that's what i tried first but didn't get the stderr output in the error. So i went for the approach where I always add the output to the debug log.

Reply:
I'm not suggesting to remove the output log, but in addition to that output, it should be obvious when there was an error/there was not an error. Not sure it will always be possible to know from just 'output'

Reply:
Right but we're returning the `err` from the call. Or do you mean, we should add more context to the error?

Reply:
Just slightly worried that printing only 'output' in debug could be misleading. For example if 'output' seems to imply the command was run ok, but it was not, an error is returned but the caller ignores the error and this causes a bug. Since we are already printing debug in that codepath, an explicit "command failed" when we return an error would not hurt imo.

Reply:
thanks for explaining. Yeah I agree.